tox.ini

```diff
@@ -38,7 +38,7 @@ deps =
 
 [testenv:mypy]
 deps =
-    mypy==0.790
+    mypy==0.982
```

Comment:
would it make sense to update directly to the latest mypy version?

Reply:
Good call, as the new version picked up some typing inconsistencies. https://github.com/scrapinghub/scrapy-poet/pull/89/commits/4881d6fa4b8163abd96cc1b97d9fde22f1066b9a